### PR TITLE
Console.error tantrums

### PR DESCRIPTION
**Pull request by jfparadis**
Using console.warn() makes messages more visible.

### CODE REVIEW

File: shim/src/utilities.js [hunk 1]
```diff
@@ -5,16 +5,18 @@
 
 export function throwTantrum(s, err = undefined) {
   const msg = `please report internal shim error: ${s}`;
+
   // note: we really do want to log these 'should never happen' things. there
   // might be a better way to convince the linter, though.
   // eslint-disable-next-line no-console
-  console.log(msg);
+  console.warn(msg);
```
Comment (ljharb) [inline shim/src/utilities.js]: console.error, if they’re things that should never happen?

Comment (jfparadis) [reply shim/src/utilities.js]: Do we need a distinction form the actual error thrown?

Comment (ljharb) [reply shim/src/utilities.js]: i'm just referring to the console message - "warn" is more for stuff that's worth fixing but not necessary to fix, "error" is for stuff that's an error - and people's console filtering settings might be set to "only error". not a big deal either way, ofc.

**Comment by jfparadis** [comment]
@ljharb Added tests yesterday.

**Pull request merged**